src/modules/administration/components/classes.jsx

```diff
@@ -0,0 +1,95 @@
+import AdminSection from './admin-section';
+import ModalForm from './modal-form';
+import Table from './table';
+
+class SectionClasses extends AdminSection {
+
+	constructor(props) {
+		super(props);
+
+		this.options = {
+			title: 'Klassen',
+			addLabel: 'Klasse hinzufügen',
+			editLabel: 'Klasse bearbeiten',
+			submitCallback: (data) => {
+				this.props.actions.updateClass(data);
+			}
+		};
+
+		this.defaultRecord = {
+			name: '',
+			schoolId: '582c58c72038900b2b7010a8' // TODO: no _id
```

Comment:
not working for me locally because I don't have this entry in my db... Maybe we need the school of the logged in user here.